docs/UserGuide.md

```diff
@@ -33,40 +36,74 @@ This user guide provides in-depth descriptions of all the available features in
+| **Command Box**                        | This is where you **enter commands**.                                                                                                                             |
+| **Result Display**                     | This is where the detailed **results** of the command will be shown.                                                                                              |
+| **Guest Card**                         | This is where all **guest information** of a guest will be shown.                                                                                                 |
+| **Status Bar**                         | This shows the location of your **save file**.                                                                                                                    |
+| ![Phone](images/phone.png)             | This is the **phone number** of the guest.                                                                                                                        |
+| ![Email](images/email.png)             | This is the **email address** of the guest.                                                                                                                       |
+| ![Room](images/room.png)               | This is the **room** the guest is staying in.                                                                                                                     |
+| ![DateRange](images/dateRange.png)     | These are the **dates of stay** of the guest.                                                                                                                     |
+| ![NumberOfGuests](images/group.png)    | This is the total **number of guests** in the guest's room.                                                                                                       |
+| ![IsRoomClean](images/isRoomClean.png) | This shows whether the guest's **room has been cleaned**. **Green** means that the room has been cleaned while **red** means that the room has yet to be cleaned. |
+| ![Bill](images/bill.png)               | This is the current **bill** of the guest.                                                                                                                        |
+| ![Request](images/request.png)         | This shows any **requests** made by the guest.                                                                                                                    |
 
 [Return to Table of Contents](#table-of-contents)
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
-1. Ensure you have `Java 11` or above installed in your Computer.
-
+1. Ensure you have `Java 11` or above installed in your Computer. <br><br>
+
+   * If you are using a Mac, open the terminal and enter `java --version`.
+   * If you are using a Windows, open the command window and enter `java -version`. 
```

Comment:
Should we teach them how to open it here as well?

Reply:
Yes I was thinking the same, whether we should show them both options of double-clicking and running `java -jar GuestBook.java`

Reply:
Yes, added!